test/acquisition/test_proximal.py

```diff
@@ -133,6 +166,16 @@ def test_proximal(self):
                     )
                     self.assertEqual(qei_prox.shape, torch.Size([4]))
 
+                    # test acquisition function with
+                    # negative values w/o softplus transform specified
+                    negative_acqf = NegativeAcquisitionFunction(model)
+                    bad_neg_prox = ProximalAcquisitionFunction(
+                        negative_acqf, proximal_weights=proximal_weights
+                    )
+
+                    with self.assertRaises(RuntimeError):
+                        bad_neg_prox(test_X)
```

Comment:
Can you use `assertRaisesRegex` in order to check for the proper error message here?